src/server/server_family.cc

```diff
@@ -470,6 +471,44 @@ bool DoesTimeMatchSpecifier(const SnapshotSpec& spec, time_t now) {
          DoesTimeNibbleMatchSpecifier(spec.minute_spec, min);
 }
 
+void ValidateSnapshotFlags() {
+  string save_time = GetFlag(FLAGS_save_schedule);
+  string snapshot_cron_exp = GetFlag(FLAGS_snapshot_cron);
+
+  if (!snapshot_cron_exp.empty() && !save_time.empty()) {
+    LOG(ERROR) << "save_time and cron_exp flags should not be set simultaneously";
+    exit(1);
+  }
```

Comment:
I'd move this `if` to inside `InferSnapshotCronExpr()` instead of reading the flags twice. Not for performance, just for consistency

Reply:
I agree and it was there at first, however it caused the exit to throw an exception that showed the stack trace which IIRC @romange wanted to avoid.
It seems that `exit(1)` inside a `Fiber` causes that...
Is there any way I can achieve a peaceful `exit` inside a `Fiber`?

Reply:
Maybe `quick_exit()` or `abort()` would not print that?
FWIW I think @romange was planning to remove that print, so perhaps it's not that bad?

Reply:
`quick_exit()` worked.
Got it back into `InferSnapshotCronExpr()`.